internal/config/plugin.go

```diff
@@ -42,11 +42,11 @@ func RunPluginFn(fn string, args ...lua.LValue) error {
 // RunPluginFnBool runs a function in all plugins and returns
```

Comment:
The comment needs an update?

Reply:
You keep punishing me for my carelessness. 😄
But you're right, it's not for every plugin any longer.